doc/migrating-from-0.x.md

```diff
@@ -38,7 +38,26 @@ Replaced by `.formatMessage( path [, variables ] )`. The new API is quite differ
 
 ## Globalize.parseInt/parseFloat
 
-Replaced by `.parseNumber( value [, options] )`.
+Replaced by `.parseNumber( value [, options] )`.  `parseNumber` is an alias for [`.numberParser( [options] )( value )`](https://github.com/jquery/globalize/blob/master/doc/api/number/number-parser.md).  So where you might have previously executed:
+
+```js
+Globalize("en").parseFloat("123,456.789")
+// > 123456.789
+```
+
+You could now execute:
+
+```js
+Globalize("en").parseNumber("123,456.789")
+// > 123456.789
+```
+
+Or: 
+
+```js
+Globalize("en").numberParser()("123,456.789")
+// > 123456.789
+```
```

Comment:
The examples is indeed a plus. But, I'm wondering if we could drop the second one `(numberParser()` for simplicity. Therefore, I would do this change:

```diff
diff --git a/doc/migrating-from-0.x.md b/doc/migrating-from-0.x.md
index 969f192..7a179b9 100644
--- a/doc/migrating-from-0.x.md
+++ b/doc/migrating-from-0.x.md
@@ -38,7 +38,7 @@ Replaced by `.formatMessage( path [, variables ] )`. The new API is quite differ
 
 ## Globalize.parseInt/parseFloat
 
-Replaced by `.parseNumber( value [, options] )`.  `parseNumber` is an alias for [`.numberParser( [options] )( value )`](https://github.com/jquery/globalize/blob/master/doc/api/number/number-parser.md).  So where you might have previously executed:
+Replaced by `.parseNumber( value [, options] )`. So where you might have previously executed:
 
 ```js
 Globalize("en").parseFloat("123,456.789")
@@ -52,12 +52,7 @@ Globalize("en").parseNumber("123,456.789")
 // > 123456.789
 \```
 
-Or: 
-
-```js
-Globalize("en").numberParser()("123,456.789")
-// > 123456.789
-```
+`parseNumber` is an alias for [`.numberParser( [options] )( value )`](https://github.com/jquery/globalize/blob/master/doc/api/number/number-parser.md).  
 
 ## Globalize.parseDate
 
```